configuration/online-telemetry.adoc

```diff
@@ -0,0 +1,54 @@
+[id="online-telemetry"]
+= Understanding online telemetry
+include::modules/common-attributes.adoc[]
+:context: online-telemetry
+
+toc::[]
+
+[role="_abstract"]
+{product-title} includes online telemetry, that is, it optionally gathers data about {product-title} and its components.
+The online telemetry data includes various operational and usage metrics, as well as information about the size, shape, and possible adverse conditions of the environments {product-title} components are running in.
+
+{product-title} does not collect node names, IP addresses, or other identifiable information about your infrastructure.
+See the xref:#collected-data[collected data] section for detailed information about the data {product-title} collects.
+
+Red Hat uses the collected data to analyze anomalies, fix problems, and improve the quality of future {product-title} versions based on real-world usage.
+{product-title} transmits online telemetry data over a TLS-encrypted connection with negligible performance and network traffic impact.
+
+[WARNING]
+====
+* If you control egress traffic from your Central deployment, you must allow outgoing connections to `telemetry.stackrox.io` on TCP port number 443.
+* If you are using {product-title} in offline mode, online telemetry does not send any data.
+However, Red Hat recommends that you to xref:../../configuration/generate-diagnostic-bundle.adoc[generate the diagnostic bundle] and attach it when you create a support case.
+====
+
+include::modules/online-telemetry-collected-data.adoc[leveloffset=+1]
+
+include::modules/inspect-online-telemetry-data.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../configuration/generate-diagnostic-bundle.adoc[Generating a diagnostic bundle]
```

Comment:
This one does work, but you need the anchor too.

Reply:
I want to reference the assembly directly because it has two methods of generating the bundle by using the API token and the admin password.

Reply:
That's fine, but we're still supposed to add the anchor for the assembly. So it should be:

```
xref:../configuration/generate-diagnostic-bundle.adoc#generate-diagnostic-bundle[Generating a diagnostic bundle]
```